mergify/src/main/scala/org/typelevel/sbt/mergify/MergifyAction.scala

```diff
@@ -57,6 +63,102 @@ object MergifyAction {
       }
   }
 
+  class RequestReviews private (
+      private val users: Option[OptionallyWeighted],
+      private val teams: Option[OptionallyWeighted],
+      private val usersFromTeams: Option[OptionallyWeighted],
+      private val randomCount: Option[Int])
+      extends MergifyAction {
+    override private[mergify] def name = "request_reviews"
+
+    private def copy(
+        users: Option[OptionallyWeighted] = users,
+        teams: Option[OptionallyWeighted] = teams,
+        usersFromTeams: Option[OptionallyWeighted] = usersFromTeams,
+        randomCount: Option[Int] = randomCount): RequestReviews =
+      new RequestReviews(users, teams, usersFromTeams, randomCount) {}
+
+    def andUsers(user: String, users: String*): RequestReviews =
```

Comment:
Assuming we like this pseudo-builder style, there's a bikeshed opportunity: "and" vs "with"

Reply:
It might also be nice to try to prevent someone from calling `andUsers` if `users` has already been set, but that might be overkill 🙂

Reply:
Wow, thanks, this looks really really nice! I do like it :)

I'm partial for "with". Since these are collections, "and" makes me thing it's appending (and not replacing). Which actually could be cool too.